addons/mail/static/src/components/attachment_image/attachment_image.xml

```diff
@@ -13,17 +13,17 @@
                     t-att-data-mimetype="attachmentImage.attachment.mimetype"
                 >
                     <t t-if="!attachmentImage.attachment.isUploading">
-                        <img class="img img-fluid" t-att-src="attachmentImage.imageUrl" t-att-alt="attachmentImage.attachment.name" t-attf-style="max-width: min(100%, {{ attachmentImage.width }}px); max-height: {{ attachmentImage.height }}px;"/>
+                        <img class="img img-fluid my-0 mx-auto" t-att-src="attachmentImage.imageUrl" t-att-alt="attachmentImage.attachment.name" t-attf-style="max-width: min(100%, {{ attachmentImage.width }}px); max-height: {{ attachmentImage.height }}px;"/>
                     </t>
                     <t t-if="attachmentImage.attachment.isUploading">
-                        <div class="o_AttachmentImageUploading d-flex align-items-center justify-content-center position-absolute" title="Uploading">
+                        <div class="o_AttachmentImageUploading d-flex align-items-center justify-content-center" title="Uploading">
                             <i class="fa fa-spin fa-spinner"/>
                         </div>
                     </t>
-                    <div class="o_AttachmentImage_imageOverlay d-flex flex-row justify-content-end position-absolute">
-                        <div class="o_AttachmentImage_actions d-flex flex-column">
+                    <div class="o_AttachmentImage_imageOverlay d-flex flex-row justify-content-end p-2 text-white opacity-0 opacity-100-hover">
+                        <div class="o_AttachmentImage_actions d-flex flex-column justify-content-between">
                             <t t-if="attachmentImage.attachment.isEditable">
-                                <div class="o_AttachmentImage_action o_AttachmentImage_actionUnlink text-center"
+                                <div class="o_AttachmentImage_action o_AttachmentImage_actionUnlink btn btn-dark opacity-75 opacity-100-hover"
```

Comment:
Also this button is now too visible I think, make sure the design is ok with FP. For v15 he requested that we made it more subtle.

Reply:
Approved, he thinks it's an improvement 👍